generic/OpenMP_Support.F90

```diff
@@ -519,20 +523,22 @@ function make_subgridcomps(GridComp, run_entry_points, num_grids, unusable, rc)
         character(len=ESMF_MAXSTR) :: comp_name
         character(len=:), allocatable :: labels(:)
         integer :: phase
+        type(ESMF_Config) :: CF
```

Comment:
I am not aware of why CF is involved in this PR.  Please explain.

Reply:
CF is retrieved in MOIST run method and therefore it is being set for the mini GCs. However it appears not to be used at all therefore it can and will be removed.

Reply:
I thought CF is not needed but it seems to be needed in some of the MOIST modules